httpserver/src/main/kotlin/tbdex/sdk/httpserver/handlers/GetExchanges.kt

```diff
@@ -62,11 +63,9 @@ suspend fun getExchanges(
     return
   }
 
-  // todo: verify JWT token using new CryptoUtils.verify() method
-  // to be written to address these issues:
-  // generating JWT token: https://github.com/TBD54566975/tbdex-kt/issues/121
-  // verifying JWT token: https://github.com/TBD54566975/tbdex/issues/210
-
+  val token = arr[1]
+  // TODO: how to access pfiDid here?
```

Comment:
@phoebe-lew so i want to actually call RequestToken.verify() in this protected endpoint, but not sure how to access pfiDid from here?

Reply:
punting it to a separate issue https://github.com/TBD54566975/tbdex-kt/issues/135